cognite/seismic/protos/types.proto

```diff
@@ -279,6 +279,8 @@ enum IngestionSource {
 }
 
 enum Dimensions {
-    TWO_DEE = 0;
-    THREE_DEE = 1;
+    INVALID_ZERO = 0; // Not used
+    ONE_DEE = 1; // Not implemented
```

Comment:
Is it possible to use the `reserved` keyword here for 0 to 1?

Reply:
I tried, it didn't seem to work

Reply:
I guess maybe not if the 0 is the default :/

Reply:
proto says no. All enums need a zero-valued enum constant. "INVALID" would be a perfectly good choice for that , though.